StormyHares/BangazonCLI
# Customer database mw

* **The Name of this branch is**
customerDatabase-mw

* **Please check if the PR fulfills these requirements**



* **What kind of change does this PR introduce?** (Bug Fix, Feature, Document Update...)
Feature


* **What Ticket is associated with this PR**
#1 #2 


* **What is the new behavior?**  (if this is a feature change)
1. when a user chooses to add a new customer in the main cli menu, the customer data is stored in the database.
2. when a user wants to select the active customer, the list of customers is created from the database and presented to the user. Once selected, the customer Id is passed to the feature menu.

* **List all core functionality** (Must include at least one Unit Test for Each)
1. Customers can be added and all customers retrieved from the database and added to a list.
2. Customers can be added and a single customer is retrieved from the database from their customer Id.

* **Other information / Steps to Test**:
1. `git fetch --all`
1. `git checkout customerDatabase-mw`
1. `dotnet test` confirm that all tests passed
1. `dotnet run`
1. enter a new customer
1. open the database in db browser and confirm that your customer data has been properly added 
1. from the main menu navigate to the list from which you can select an active customer
1. confirm that the customer you added is listed
